tests/kernel/qa_test_klp.pm

```diff
@@ -36,8 +41,11 @@ sub run {
     (is_sle(">12-sp1") || !is_sle) ? $self->select_serial_terminal() : select_console('root-console');
 
     add_suseconnect_product("sle-sdk") if (is_sle('<12-SP5'));
-    zypper_call('in -l autoconf automake gcc git make');
-
+    if (is_transactional) {
+        trup_call('in -l autoconf automake gcc git make');
```

Comment:
Shouldn't the argument start with `pkg in`?

Reply:
You're right, I'll fix it